kernel/dev/vga/graphics.cpp

```diff
@@ -1,12 +1,15 @@
 /**
  * @file graphics.cpp
  * @author Keeton Feavel (keetonfeavel@cedarville.edu)
+ * Michel (JMallone) Gomes (michels@utfpr.edu.br)
```

Comment:
@Kfeavel correct me if I'm wrong, but I *think* you need another `@author` prefix before your name.

Reply:
I think we've been inconsistent about it, but yeah, we should prefix it with the author tag.

Reply:
i saw in another document without this prefix, but now is right :)

Reply:
Yeah, this isn't your fault or anything. We have been inconsistent with code style and a lot of other stuff. Trying to be better about it now, so thank you for your cooperation. 🙂

Reply:
Little by little the house is being organized ;D